packages/design/README.stories.mdx

```diff
@@ -0,0 +1,83 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Design/README" />
```

Comment:
Last one, this is actually under packages https://atlantis.getjobber.com/packages/design

Reply:
should it be `Packages/Design/{page}`?

Reply:
Nope. just Packages/Design since it shows up like this. 
<img width="284" alt="image" src="https://user-images.githubusercontent.com/15986172/219669783-9a83b37b-5c85-4a4e-b39e-339ec86d7f6a.png">

this file is called readme so Github we also reads it https://github.com/GetJobber/atlantis/tree/master/packages/design